pyproject.toml

```diff
@@ -10,4 +10,4 @@ git_describe_command = "git describe --dirty --tags --long --match *[0-9].[0-9]*
 
 [tool.black]
 line-length = 120
-target-version = ['py27']
+target-version = ['py36']
```

Comment:
This definitely shouldn't be changed

Reply:
Hi @chrisburr 

Just curious, why? :)

More seriously, black (v22.8) on my machine (arch linux) refused to work with the given target version of py27. And once I made the change to get black working, git refused to commit my code, unless I also committed pyproject.toml also.

So, I was left with a catch-22 situation. I am happy for you to ignore the commit to this file if this change is such a breaking change (is it?).

Cheers,
Raja.

Reply:
Well as you can see the tests on Github Actions did not run, so yes this is breaking.

Reply:
> Just curious, why? :)

Some DIRAC installations are still using Python 2.7 and using a newer black version causes the code to be reformatted in a way which causes a `SyntaxError` to be raised. This is why we still use the old `black` version in `rel-v7r2`/`rel-v7r3`. It's already been updated to be `py39` for `rel-v8r0`/`integration`.

Reply:
Hi @fstagni and @chrisburr ,

Thanks for the explanations. With the latest two commits, I hope I have reset the two files - pyproject and pre-commit-config. The AREXComputingElement remains unchanged.

Let me know if there is something else that is going wrong here, as I still do not see all the checks running.

Cheers,
Raja.